Python/cp/anagram_checker.py

```diff
@@ -1,40 +1,21 @@
 """
 This script takes two input strings and compare them to check if they are anagrams or not.
 """
+def mysort(s):    	#function that splits the letters
```

Comment:
@chicken-biryani Do not remove the old program instead keep both the program in the same file and add a proper comment in your program about the function you have used.